app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topics-controllers');
+const app = express()
+
+app.use(express.json())
```

Comment:
We haven't made a post request yet so this line should be removed 🔍 
It's important to only include code that meets the need of the current ticket/task